### ISSUE TITLE
seo-image dynamic url?

### ISSUE THREAD

**Issue by miyamoto-san** [GuillaumeLeclerc/vue-seo#4]
Hey,

How would I go about binding a variable to the url param?
`<seo-image url="{{var}}" width="600" height="500"></seo-organization>`

For example doesn't work!

**Comment by GuillaumeLeclerc** [comment]
`<seo-image :url="var" width="600" height="500"></seo-image>`

Should be good but check your markup you are closing an organization instead of the image

**Comment by miyamoto-san** [comment]
Ok, the guide should be updated then

`<seo-image
  url="http://company.com/image.png"
  width="300"
  height="500"

> </seo-organization>`

I will give it a try tomorrow when I am back with the code, I'll be reporting back!

**Comment by GuillaumeLeclerc** [comment]
There is indeed a typo in the docs I'm sorry about that

**Comment by miyamoto-san** [comment]
Yeah, no worries I did try what you suggest using v-bind but it could be I didn't try to fix the typo described above!

**Comment by GuillaumeLeclerc** [comment]
Is it working fine or do you still have a problem ?

**Comment by miyamoto-san** [comment]
I am not able to check it right now but I commit to reporting back no later than tomorrow!
